### PR TITLE
add new fields

### PR DESCRIPTION
**Pull request by bradleyramos**
Wasn't able to test with backend but followed template and matched Rhee's element names

### CODE REVIEW

File: src/components/Upload.js [hunk 1]
```diff
@@ -35,7 +41,12 @@ const formErrors = {
   link: false, 
   category: false, 
   dataType: false, 
-  description: false
+  description: false,
+  keyTerms: false,
+  primaryAudience: false,
+  purpose: false,
+  decisions: false,
+  residentExpert: false,
```
Comment (alexr17) [inline src/components/Upload.js:49]: Are trailing commas okay in js?

File: src/components/Upload.js [hunk 2]
```diff
@@ -128,7 +139,12 @@ const Upload = ({ modalOpen, setModalOpen, setUploadConfirmed }) => {
       link: handleEmpty(request.metadata.raw_source_link), 
       category: handleEmpty(request.category_ids), 
       dataType: handleEmpty(request.datatype_id),
-      description: handleEmpty(request.metadata.description)
+      description: handleEmpty(request.metadata.description),
+      keyTerms: handleEmpty(request.metadata.key_terms),
+      primaryAudience: handleEmpty(request.metadata.primary_audience),
+      purpose: handleEmpty(request.metadata.purpose),
+      decisions: handleEmpty(request.metadata.decisions),
+      residentExpert: handleEmpty(request.metadata.resident_expert),
```
Comment (alexr17) [inline src/components/Upload.js:147]: Another trailing comma

File: src/components/Upload.js [hunk 3]
```diff
@@ -239,30 +255,42 @@ const Upload = ({ modalOpen, setModalOpen, setUploadConfirmed }) => {
...
+              label="Who can provide context around the dataset and answer questions about it?"
+              required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, resident_expert: e.target.value}})}
+              error={errors.residentExpert}
             />
             <Form.Field
               label="Where does this dataset show up (i.e. in major reports)?"
               control={TextArea}
               //required
+              onChange={(e) => setRequest({...request, metadata: {...request.metadata, appearances: e.target.value}})}
```
Comment (alexr17) [inline src/components/Upload.js:293]: No error here?

Comment (bradleyramos) [reply src/components/Upload.js:293]: no error because this is the only field we are not requiring

**Pull request merged**